.editorconfig

```diff
@@ -1,5 +1,5 @@
-# top-most EditorConfig file
```

Comment:
Are the changes in this editorconfig supposed to be a part of this change?

Reply:
Some accident. Those changes have been reverted